kinds/rolebinding/role_binding_kind.cue

```diff
@@ -0,0 +1,46 @@
+package kind
+
+name:        "RoleBinding"
+maturity:    "merged"
+description: "Role bindings link a set of subjects to a configured role"
+
+// ?? Writing a role binding requires "write" permission (or *)
+// on the parent roleRef 
+
+lineage: seqs: [
+	{
+		schemas: [
+			// 0.0
+			{
+				spec: {
+					// The role we are discussing
+					roleRef: #BuiltinRoleRef | #CustomRoleRef
+
+					// The set of subjects who share the same role
+					// ??? this is a list in k8s... should it be in grafana?
+					// as a list it implies that ability to edit a role+role_binding
+					// meas you can see all the teams+users that use it.
+					// -- is that OK, accurate?
+					subjects: [...#RoleBindingSubject]
```

Comment:
@DanCech -- do you think this should be a single subject, or a list?  The main issue is who can edit the resource that defines this mapping.

Using a single subject<>role means we don't have to think too hard what the security implications are on having multiple subjects listed for the role.  BUT it would be a lot more verbose.

With a single subject, we would likely want a unique constraint on roleRef+subject, with multiple it would be just on roleRef

Reply:
Probably more straightforward to keep it as a single element and sidestep the issue